src/main/java/randoop/operation/TypedClassOperation.java

```diff
@@ -158,6 +159,42 @@ public boolean hasWildcardTypes() {
     return new ArrayList<>(paramSet);
   }
 
+  /**
+   * Returns the fully-qualified signature for this operation if it is a method or constructor call.
+   *
+   * @return this operation's fully qualified signature if it is a method or constructor call, null
+   *     otherwise.
+   */
+  public String getFullyQualifiedSignature() {
```

Comment:
I see a lot of similarities between the code of this method and that of getRawSignature which appears directly below it.
Could you use the RawSignature object to avoid some code duplication (at the cost of slightly less efficient creation of the string, but that isn't a huge deal)?

Reply:
Hmm it's not immediately obvious to me how we should proceed (only four lines are duplicated and each field with the exception of classPackaged is referenced somewhere following the four lines).